db/hybrid_logical_vector.go

```diff
@@ -283,3 +304,11 @@ func (hlv *HybridLogicalVector) computeMacroExpansions() []sgbucket.MacroExpansi
 	}
 	return outputSpec
 }
+
+// addToPreviousVersions will take a source/version pair and add it to the HLV previous versions map
+func (hlv *HybridLogicalVector) addToPreviousVersions(source string, version uint64) {
+	if hlv.MergeVersions == nil {
+		hlv.MergeVersions = make(map[string]uint64)
+	}
+	hlv.MergeVersions[source] = version
```

Comment:
This looks like it's incorrectly targeting hlv.MergeVersions (instead of hlv.PreviousVersions)?